src/components/ProfitAndLossSummaries/internal/ProfitAndLossSummariesSummary.tsx

```diff
@@ -25,27 +27,59 @@ export function ProfitAndLossSummariesSummary({
   label,
   amount,
   isLoading,
+  percentChange,
+  comparisonMonth,
+  isExpense = false,
   slots,
   variants,
 }: ProfitAndLossSummariesSummaryProps) {
   const { Chart } = slots ?? {}
-  const { size = '2xs' } = variants ?? {}
 
-  const dataProperties = toDataProperties({ size })
+  const showPercentChange = percentChange !== undefined && percentChange !== null && comparisonMonth
+
+  const isGoodChange = showPercentChange && (isExpense ? percentChange < 0 : percentChange >= 0)
+  const arrow = showPercentChange && percentChange >= 0 ? '↑' : '↓'
 
   return (
-    <div className={CLASS_NAME} {...dataProperties}>
-      {Chart ? <div className={CHART_AREA_CLASS_NAME}>{Chart}</div> : <div className={CHART_AREA_EMPTY_FRAME_CLASS_NAME} />}
-      <ProfitAndLossSummariesHeading variants={variants}>
-        {label}
-      </ProfitAndLossSummariesHeading>
-      {isLoading
-        ? (
-          <SkeletonLoader height='20px' />
-        )
-        : (
-          <MoneySpan slot='amount' amount={amount} size='lg' bold />
-        )}
-    </div>
+    <HStack gap='xs' align='center'>
+      {Chart && (
+        <HStack
+          align='center'
+          justify='center'
+          className={CHART_AREA_CLASS_NAME}
+        >
+          {Chart}
+        </HStack>
+      )}
+
+      <HStack gap='xs' fluid pi='sm'>
+      <VStack gap='3xs' fluid>
+        <ProfitAndLossSummariesHeading variants={variants}>
+          {label}
+        </ProfitAndLossSummariesHeading>
+        {isLoading
+          ? (
+            <SkeletonLoader height='20px' />
+          )
+          : (
+            <MoneySpan slot='amount' amount={amount} size='lg' bold />
+          )}
+      </VStack>
+
+      <VStack gap='3xs' align='end' hidden={!showPercentChange}>
+        <HStack gap='3xs' align='center'>
+          <Span size='md' weight='bold' status={isGoodChange ? 'success' : undefined}>
+            {arrow}
+          </Span>
+          <Span size='md' weight='normal' status={isGoodChange ? 'success' : undefined}>
+            {formatPercentageChange(percentChange ?? null)}
+          </Span>
+        </HStack>
+        <Span size='xs' variant='subtle' noWrap>
+          vs. {comparisonMonth || ''}
+        </Span>
+      </VStack>
+      </HStack>
```

Comment:
Any reason why we're visually hiding the div with CSS instead of:
```suggestion
      {showPercentChange &&  (
      <VStack gap='3xs' align='end' hidden={!showPercentChange}>
        <HStack gap='3xs' align='center'>
          <Span size='md' weight='bold' status={isGoodChange ? 'success' : undefined}>
            {arrow}
          </Span>
          <Span size='md' weight='normal' status={isGoodChange ? 'success' : undefined}>
            {formatPercentageChange(percentChange ?? null)}
          </Span>
        </HStack>
        <Span size='xs' variant='subtle' noWrap>
          vs. {comparisonMonth || ''}
        </Span>
      </VStack>
      </HStack>)
    }
```

Reply:
Yes, I originally had it that way, but the height of the percentage change and the other text is different by like 2px, so when you click back and forth between months that do or do not show the % change, it causes a flicker where the text repositions. I do not like that. I opted for this as opposed to maybe adding extra padding to even it out since that felt more fragile. Open to changing!